cpp/test/IceSSL/configuration/SchannelTests.cpp

```diff
@@ -299,6 +350,7 @@ serverValidatesClientSettingTrustedRootCertificates(Test::TestHelper* helper, co
                 CertDuplicateCertificateContext(serverCertificate);
                 return serverCertificate;
             },
+            .clientCertificateRequired = true,
```

Comment:
Should this have been failing before?

Reply:
No. We can improve the test to ensure the client provided a certificate. For a follow-up PR.